public_html/js/mpform.js

```diff
@@ -0,0 +1,79 @@
+//jQuery time
+var current_fs, next_fs, previous_fs; //fieldsets
+var left, opacity, scale; //fieldset properties which we will animate
+var animating; //flag to prevent quick multi-click glitches
+
+$(".next").click(function(){
+	current_fs = $(this).parent().parent();
+	next_fs = $(this).parent().parent().next();
+	var valid = PIN.Form.validatePage($(this).parent().prev().attr('id'))
+	if(valid){
```

Comment:
That kind of DOM selector chaining can be very brittle if the HTML structure ever changes. Is it possible to identify with a unique id and/or class combination?